docs/core_modules/data_modules/documents_and_nodes/usage_documents.md

```diff
@@ -34,7 +34,7 @@ This section covers various ways to customize `Document` objects. Since the `Doc
 
 ### Metadata
 
-Documents also offer the chance to include useful metadata. Using the `metadata` dictionary on each document, additional information can be included to help inform responses and track down sources for query responses. This information can be anything, such as filenames or categories. If you are intergrating with a vector database, keep in mind that some vector databases require that the keys must be strings, and the values must be flat (either `str`, `float`, or `int`).
+Documents also offer the chance to include useful metadata. Using the `metadata` dictionary on each document, additional information can be included to help inform responses and track down sources for query responses. This information can be anything, such as filenames or categories. If you are integrating with a vector database, keep in mind that some vector databases require that the keys must be strings, and the values must be flat (either `str`, `float`, or `int`).
```

Comment:
@SridharCR we actually have a tool `codespell` (configured in `pyproject.toml`) added that can automatically check spelling in `docs/`.  If you're interested, feel free to uncomment its `docs/` disable and fix many other instances of typos

Reply:
@jamesbraza Yeah sure we'll do the needful on the same PR

Reply:
Yeah there are `codespell` flags `--interactive` and `--write-output` to help fix the errors in `docs/` automatically, if wanted

Reply:
@jamesbraza Did relevant changes with codespell, got some issues with ipynb files, where in the encoded strings are being checked, hence enabled docs and disabled ipynb files. Kindly review and let me know if anything else is requried

Reply:
Oh yeah the encoded strings are tough haha, can't solve with `ignore-words-list`.

What do you think of keeping `codespell` enabled for `.ipynb` (since they're heavily used for docs), and instead adding the exact IPython notebooks to `skip`?  Something like:

```toml
skip = "./.git,./*venv,./examples,./experimental,docs/some/notebook1.ipynb,docs/some/notebook2.ipynb,*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,"
```

Also, if you think that's a bad idea, I am okay with universally disabling for `.ipynb`. This PR is still a step forward, so thanks!

Reply:
Yeah that makes sense, I think for now, let's merge this and we'll open new issue where I'll fix all the ipynb files